packages/shorebird_cli/README.md

```diff
@@ -324,6 +324,22 @@ Would you like to continue? (y/N) Yes
 ✅ New Channel Created!
 ```
 
+### Cancel Subscription
+
+Cancel your Shorebird subscription using `shorebird subscription cancel` command:
+
+```bash
+shoreburd subscription cancel
+```
+
+**Sample**
+
+```bash
+$ shorebird subscription cancel
+This will cancel your Shorebird subscription. Are you sure? (y/N) Yes
```

Comment:
This is GREAT.  But we should be explicit about what happens (as you've asked in inc).  e..g

```
Note: Your access to Shorebird will continue until $DATE after which all data stored with Shorebird will be deleted as per our privacy policy: https://shorebird.dev/privacy.html

Apps on devices you've built with Shorebird will continue to function normally, but will not receive further updates.
```

We could get fancier in the future explaining which apps that means, etc.  But honestly this looks great, even as-is.

Reply:
Have we thought about how they might re-subscribe (that can be a later patch).  Of course once we have "re-subscribe" we then have "subscribe" in the first place. :D

Reply:
Can we update the README to include a sample of the new output format?